api/models/User.js

```diff
@@ -17,7 +17,6 @@ module.exports = bookshelf.Model.extend(merge({
   comments: function () {
     return this.hasMany(Comment)
     .query(q => {
-      q.join('posts', 'posts.id', 'comments.post_id')
```

Comment:
and we are sure this doesnt break comments anywhere else?

Reply:
I am as sure as I can be in having tested through the UI comment creation and presentation on Posts. No issues arose there. I wouldn't expect any trouble, but I'm confused why this extra and self-apparently redundant `join` was added in the first place. I have to for now assume it was added in error.